fix/mab2De-Sol1Holdings_seq2strapi.fix

```diff
@@ -1,114 +1,114 @@
+
+  # Identifier for the local holding record.
+	copy_field("001.a", "holdingId")
+
+
+  # Old date history for documentation purposes.
+	copy_field("002a.a", "dateCreated")
+	copy_field("003.a", "dateModified")
+
+  # Identifiers mapped from referenced hbz id in 012.
+	copy_field("012.a", "hbzId")
+	copy_field("012.a", "almaMmsId")
+	copy_field("012.a", "zdbId")
+
+  # Other holding related information combined into `holdingInfo`
+	do list(path:"210a", "var":"$i")
+		copy_field("$i.d", "@210a.$append")
+		copy_field("$i.n", "@210a.$append")
 		if is_array("$i.k")
-			join_field("$i.k",", ")
+			join_field("$i.k", ", ")
 		end
-		copy_field("$i.k","@210a.$append")
+		copy_field("$i.k", "@210a.$append")
 		if is_array("$i.j")
-			join_field("$i.j",", ")
+			join_field("$i.j", ", ")
 		end
-		copy_field("$i.j","@210a.$append")
-		join_field("@210a",", ")
-		paste("holdingInfo.$append","~210a: ", "@210a")
+		copy_field("$i.j", "@210a.$append")
+		join_field("@210a", ", ")
+		paste("holdingInfo.$append", "~210a: ", "@210a")
 	end
-	do list(path:"210b","var":"$i")
-		copy_field("$i.d","@210b.$append")
-		copy_field("$i.n","@210b.$append")
+	do list(path:"210b", "var":"$i")
+		copy_field("$i.d", "@210b.$append")
+		copy_field("$i.n", "@210b.$append")
 		if is_array("$i.k")
-			join_field("$i.k",", ")
+			join_field("$i.k", ", ")
 		end
-		copy_field("$i.k","@210b.$append")
+		copy_field("$i.k", "@210b.$append")
 		if is_array("$i.j")
-			join_field("$i.j",", ")
+			join_field("$i.j", ", ")
 		end
-		copy_field("$i.j","@210b.$append")
-		join_field("@210b",", ")
-		paste("holdingInfo.$append","~210b: ", "@210b")
+		copy_field("$i.j", "@210b.$append")
+		join_field("@210b", ", ")
+		paste("holdingInfo.$append", "~210b: ", "@210b")
 	end
-	call_macro("moveToNote",field:"220",targedField:"holdingInfo.$append")
-	join_field("holdingInfo","\n")
+	call_macro("keep_source_field_info_and_move_to", field:"220", targetField:"holdingInfo.$append")
+	join_field("holdingInfo", "\n")
 
-else
-	copy_field("001.a","item.itemId")
-	copy_field("002a.a","item.dateCreated")
-	copy_field("003.a","item.dateModified")
-	copy_field("004.a","item.dateAustausch")
-# 010.a
-# 012.a
-	# copy_field("012.a","hbzId")
-	copy_field("012.a","almaMmsId")
-	copy_field("012.a","zdbId")
-	lookup("zdbId", "hbzId2zdbId",delete:"true")
-
-	copy_field("100.a","item.callNumber")
-
-	copy_field("115.a","item.inventoryNumber")
-
-	call_macro("moveToNote",field:"125a",targedField:"item.note.$append")
-	call_macro("moveToNote",field:"125b",targedField:"item.note.$append")
-	call_macro("moveToNote",field:"107",targedField:"item.note.$append")
-	call_macro("moveToNote",field:"120",targedField:"item.note.$append")
-	call_macro("moveToNote",field:"105",targedField:"item.note.$append")
-	join_field("item.note","\n")
+else # Transformation for the ME records
+
+  # Item Id needed for the creation of the lobid item id
+	copy_field("001.a", "item.itemId")
+
+  # Old dates for documentation puprosses
```

Comment:
Typo, should be 'purposes'.

Also, unlike the rest, all these new comments are indented with (two) spaces instead of tabs.